x-pack/legacy/plugins/siem/yarn.lock

```diff
@@ -0,0 +1,3247 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
```

Comment:
are the changes to `yarn.lock` expected?

Reply:
No! Not sure how this got in, I’ll stash it. Thanks